packages/react-examples/src/react/Button/Button.IconWithTooltip.Example.tsx

```diff
@@ -35,8 +35,9 @@ export const ButtonIconWithTooltipExample: React.FunctionComponent<IButtonExampl
         id={tooltipId}
         calloutProps={calloutProps}
         styles={hostStyles}
+        setAriaDescribedBy={false}
       >
-        <IconButton iconProps={emojiIcon} title="Emoji" ariaLabel="Emoji" disabled={disabled} checked={checked} />
+        <IconButton iconProps={emojiIcon} aria-describedby={tooltipId} disabled={disabled} checked={checked} />
```

Comment:
```suggestion
        <IconButton iconProps={emojiIcon} aria-label="Emoji" disabled={disabled} checked={checked} />
```

One small change request -- using `aria-describedby` and removing the `aria-label` will mean the button will have a description, but no accessible name. In this case, the tooltip is functioning as a name, so `aria-labelledby` or `aria-label` would be better.

@ecraig12345 I think maybe it was because the tooltip isn't rendered to the DOM all the time, which caused a bug with VoiceOver not updating when it was focused? I think I handed that bug off to someone (maybe even @spmonahan? 😂), but for now it means that `tooltipId` as a label/description won't work all the time. Even aside from VoiceOver, if someone is using their screen reader in a way that does not force keyboard focus to follow the SR cursor, `tooltipId` won't work since the tooltip isn't visible.

If we do fix that tooltip bug by forcing the tooltip to always be in the DOM, then `aria-labelledby` would also be a good solution here.

Reply:
To clarify, because we have the VoiceOver bug (and, yes, it is assigned to me :)) you're recommending we use `aria-label` and because of that the `aria-label` value should be the same should be the same as the tool tip content?

Reply:
yup, exactly! For most icon buttons where the tooltip is what the button text would be if it were visible, the desired outcome is for the button's accessible name to match the tooltip text. Essentially this use case is the main reason the v9 Tooltip has a required `relationship` prop instead of defaulting to `description` 😄